stacks/generateAnImageUsingReplicate/index.ts

```diff
@@ -0,0 +1,21 @@
+import Replicate from "replicate";
+/**
+ * Brief: Generate an image usiung replicate
+ */
+
+type inputType = {
+    prompt: string
+
+}
+type modelType = `${string}/${string}:${string}`
+export default async function generateImageUsingReplicate({ model, input }: { model: modelType, input:inputType }): Promise<any> {
```

Comment:
Can you extract a typescript type from it please ?
interface ModelInfo {
   model: modelType
   input: inputType
}